platform/android/plugin/godot_plugin_jni.cpp

```diff
@@ -119,17 +119,19 @@ JNIEXPORT void JNICALL Java_org_godotengine_godot_plugin_GodotPlugin_nativeEmitS
 	String signal_name = jstring_to_string(j_signal_name, env);
 
 	int count = env->GetArrayLength(j_signal_params);
-	const Variant *args[count];
+	Variant args[VARIANT_ARG_MAX];
 
-	for (int i = 0; i < count; i++) {
+	ERR_FAIL_COND_MSG(count > VARIANT_ARG_MAX, "Maximum argument count exceeded!");
+
+	for (int i = 0; i < MIN(count, VARIANT_ARG_MAX); i++) {
 
 		jobject j_param = env->GetObjectArrayElement(j_signal_params, i);
-		Variant variant = _jobject_to_variant(env, j_param);
-		args[i] = &variant;
+		if (j_param)
+			args[i] = _jobject_to_variant(env, j_param);
```

Comment:
Same thing with the missing brackets in this PR :)

Reply:
Ah, I thought the new code style only applies from 4.0 onwards, fixed :)

Reply:
Thanks! And I guess it's true, I didn't think about that. In any case it's probably best to match the code between these 2 PRs as much as possible.

Reply:
The new code style is enforced in the `master` branch, but indeed there's no reason not to follow it when adding new code in `3.2` too.

Reply:
alright :)